xml/remote_administration.xml

```diff
@@ -255,7 +255,7 @@ nodeattr [-f genders] -k
   </important>
  </sect1>
  <sect1 xml:id="sec-remote-munge">
-  <title><emphasis>munge</emphasis> Authentication</title>
+  <title><emphasis>munge</emphasis> authentication</title>
   <para>
    <emphasis>munge</emphasis> allows for a secure communication between
```

Comment:
Since the very next paragraph capitalizes "Munge", why not here too?

Reply:
I have fixed that same issue in the release notes already. In the documentation, I'd leave it up to you, since it's a bit out of scope for the title fixes.

Reply:
My attitude is that when I noticed things, I fix them, whether it is in scope or not. If you don't want to, though, that's fine.